Yoast/javascript
# Stem regular Dutch participles

## Summary

This PR can be summarized in the following changelog entry:

* Stems regular Dutch past participles

## Test instructions

This PR can be tested by following these steps:

* Add more examples to the specs. The cases that need to be tested are: 

1) regular participles without a prefix (e.g. _gedanst_),
2) regular participles with a separable prefix, i.e. one that is followed by _ge_ (e.g. _**af**gezet_)
3) regular participles with an inseparable prefix, i.e. one that does not require _ge_ in front of it (e.g. _**be**antwoord_)
4) words that are non-participle exceptions (are on the nonParticiples exception list **or** begin with ge- and end with -heid/-tijd/-teit),
5) Participles with a prefix that is normally separable, but in this word acts as inseparable. For example, the prefix _aan_ is usually separable, but in the participle _aanschouwd_ it is inseparable  (see the inseparableCompoundVerbs array in morphology data file for the full list of those participles). 
6) Participles that are the same as the stem of the word, e.g. _omlijst_. See inseparableCompoundVerbsNotToBeStemmed in morphology data for full list.

This issue aims to cover mainly regular participles, so irregular participles falling outside of the checks described in points 5 and 6 may not (and don't have to be, for now) stemmed correctly.

## Quality assurance


Fixes #384
